src/NServiceBus.Transport.RabbitMQ/Administration/IBrokerVerifier.cs

```diff
@@ -0,0 +1,13 @@
+﻿#nullable enable
+
+namespace NServiceBus.Transport.RabbitMQ.Administration;
+
+using System.Threading.Tasks;
+using System.Threading;
+
+interface IBrokerVerifier
```

Comment:
I don't really see the benefit in having this interface. Is there a specific reason to have it?